cli/tools/test_runner.rs

```diff
@@ -338,9 +338,10 @@ pub async fn run_tests(
     let lines_regex = Regex::new(r"(?:\* ?)(?:\# ?)?(.*)")?;
 
     for specifier in &doc_modules {
+      let mut fetch_permissions = Permissions::allow_all();
```

Comment:
Way outside of scope but awkward that check_permissions was changed to mutable ref